02_configure_host.sh

```diff
@@ -125,31 +125,12 @@ if [ "$MANAGE_BR_BRIDGE" == "y" ] ; then
     fi
 fi
 
-# Add firewall rules to ensure the image caches can be reached on the host
-for PORT in 80 5000 ; do
-    if [ "${RHEL8}" = "True" ] ; then
-        sudo firewall-cmd --zone=libvirt --add-port=$PORT/tcp
-        sudo firewall-cmd --zone=libvirt --add-port=$PORT/tcp --permanent
-    else
-        if ! sudo iptables -C INPUT -i provisioning -p tcp -m tcp --dport $PORT -j ACCEPT > /dev/null 2>&1; then
-            sudo iptables -I INPUT -i provisioning -p tcp -m tcp --dport $PORT -j ACCEPT
-        fi
-        if ! sudo iptables -C INPUT -i baremetal -p tcp -m tcp --dport $PORT -j ACCEPT > /dev/null 2>&1; then
-            sudo iptables -I INPUT -i baremetal -p tcp -m tcp --dport $PORT -j ACCEPT
-        fi
```

Comment:
So I think the problem is the ansible implementation assumes the provisioning host ports are only accessible via the provisioning interface, which isn't yet up:

https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/firewall/tasks/iptables.yaml#L38

I guess we can open that up to both networks in the role, or I wonder if perhaps we should just have a per-network list of ports in the role defaults, instead of lists categorized by their usage?